app/src/editor/canvas/components/ModuleSearch.pcss

```diff
@@ -175,5 +188,14 @@
   opacity: 0;
 }
 
-global(.dragHandle) {
+.dragHandle {
+}
+
```

Comment:
Note this will change the style for all other use of react-resizable as well. I think if you pass a className to resizableBox under the draggableOpts prop, you should be able to use a locally defined class

Reply:
Alternatively, limit the scope of the selector by nesting it under `.ModuleSearch`